### PR TITLE
Imas testing msr

### PR DESCRIPTION
**Pull request by MireilleSchneider**
I make this pull request to IMAS_testing and not master, because then it is easier to see what I have modified compared to your own version.

### CODE REVIEW

File: config_iter_sdcc.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# Need to remove the stack limit to avoid segmentation fault inside codes
+ulimit -Ss unlimited
+
+# Libraries needed for code parameters
+module load XMLlib/3.3.1-intel-2020b
+
+# EXTEND PYTHON PATH AND AVOID DOUBLONS
+export PYTHONPATH=$ACTOR_FOLDER:$PYTHONPATH
+export PYTHONPATH="$(perl -e 'print join(":", grep { not $seen{$_}++ } split(/:/, $ENV{PYTHONPATH}))')"
+
```
Comment (AreWeDreaming) [inline config_iter_sdcc.sh:27]: Is ITER using Lmod to manage modules? If so I would convert that into a module. We could put ECRad into a public folder and then make it available to people with
```
module load ECRad
```

File: src/ECRad_actor_IMAS.f90 [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    call set_ece_ECRad_IMAS(ece_in, 1, output_flag, output_message)
+    call initialize_ECRad_IMAS(equilibrium, 1, output_flag, output_message)
+    call make_rays_ECRad_IMAS(core_profiles, 1)
+    if(output_flag.eq.0) call make_dat_model_ECRad(core_profiles,1, ece_out)
+
+  end subroutine ECRad_actor_IMAS
+
+end module mod_ECRad_actor_IMAS
+
+
```
Comment (AreWeDreaming) [inline src/ECRad_actor_IMAS.f90:33]: This looks like a duplicate of `mod_ECRad_actor_IMAS.f90`

**Review by AreWeDreaming — CHANGES_REQUESTED** [review]

**Comment by AreWeDreaming** [comment]
@MireilleSchneider Thank you very much for this contribution. This is quite a bit more work than I expected, so I am thankful you took that off my hands. I will investigate the error of the test.

File: .gitignore [hunk 3]
```diff
@@ -1,37 +1,7 @@
...
-src/ripple3d.mod
-!*.toml
-!*.cfg
+*.mod
+*.a
+*.o
+*~
+a.out
+__pycache__/
+bin/
```
Comment (AreWeDreaming) [inline .gitignore:7]:
```suggestion
id
MANIFEST.in
```

File: config_iter_sdcc.sh (hunk 1 above)
Comment (MireilleSchneider) [reply config_iter_sdcc.sh:27]: I don't know exactly how modules are managed on sdcc, but clearly the plan is to have a module for each code. So having an ECRad module makes sense. We have for example a CASPER module, which is another synthetic diagnostic.

File: src/ECRad_actor_IMAS.f90 (hunk 2 above)
Comment (MireilleSchneider) [reply src/ECRad_actor_IMAS.f90:33]: You are right, I thought I renamed it but I duplicated it.

Comment (MireilleSchneider) [reply src/ECRad_actor_IMAS.f90]: I have now removed the file

**Comment by AreWeDreaming** [comment]
Close because #6 includes all changes and is ahead of this.

**Pull request closed without merging**